src/components/MyKiva/BadgeModalContentJourney.vue

```diff
@@ -8,10 +8,10 @@
 				:lender-name="loan.name"
 				:lender-image-url="loan.image.url"
 				:class="{ 'tw--ml-2.5': i > 0 }"
-				:style="{ 'z-index': journeyLoans.length }"
+				:style="{ 'z-index': journeyTotalLoans }"
 			/>
-			<p v-if="journeyLoans.length - 3 > 0">
-				+{{ journeyLoans.length - 3 }}
+			<p v-if="extraLoanCount">
+				+{{ extraLoanCount }}
```

Comment:
Sorry not specifying, I meant this: https://github.com/kiva/ui/blob/e568547f9620b2eadd89d4cc231c633fef0d2e46/src/pages/Portfolio/MyKiva/MyKivaPage.vue#L306

Reply:
NVM, just related to specific journey.